src/test/unit/mocks/sjp-public-list.json

```diff
@@ -223,6 +223,70 @@
                     ]
                   }
                 ]
+              },
```

Comment:
Spotted one thing on this. Posting this file into DM causes validation errors , looks like town / county are missing for addresses. Could you add these in?

Reply:
Have made the change :)

Reply:
Thanks :)